controller/test/blower_fsm/blower_fsm_test.cpp

```diff
@@ -16,23 +16,37 @@ limitations under the License.
 #include "blower_fsm.h"
 
 #include "hal.h"
+#include "gmock/gmock-matchers.h"
 #include "gtest/gtest.h"
 
+namespace {
+
 class BlowerFsmTest : public ::testing::Test {
 public:
   BlowerFsmTest() = default;
   void SetUp() override { blower_fsm_init(); }
 };
 
+// One could also accopmlish this with gmock matchers, but wow is this easier.
```

Comment:
Could also accomplish this with a comparator + overloaded ostream operator<< :) but this is ok too.